src/base/b-content-switcher/b-content-switcher.ts

```diff
@@ -285,6 +286,17 @@ export default class bContentSwitcher extends iBlock implements iObserveDOM {
 		iObserveDOM.onDOMChange(this, records);
 	}
 
+	/**
+	 * Warns that the component is deprecated
+	 */
+	@hook('created')
+	protected deprecatedMessage(): void {
```

Comment:
Функция совершает действие, нужно что-то вроде
```suggestion
	protected showDeprecatedMessage(): void {
```